ote_sdk/ote_sdk/usecases/exportable_code/demo/demo_package/utils.py

```diff
@@ -69,6 +70,8 @@ def create_model(model_path: Path, config_file: Optional[Path] = None) -> Model:
         importlib.import_module(".model", "demo_package")
     except ImportError:
         print("Using model wrapper from Open Model Zoo ModelAPI")
+    if isinstance(parameters["model_parameters"]["labels"], dict):
+        parameters["model_parameters"]["labels"] = []
```

Comment:
What's happening here?

Reply:
I provided empty labels for model wrappers, because labels used only in converters.